src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md

```diff
@@ -14,58 +14,58 @@ order: 5
   * explain how spottable works (onMouseOver, onMouseLeave, onClick, onKeyPress, mouse emulation)
 -->
 
-In the [last step](../panels/), we focused on the app structure with Panels. Next, we'll investigate how custom and native events are used to notify your app of user actions, how state is defined for the app, and how to use events to update the state of your app.  We'll wrap up with a brief discussion on how to manage state data.
+In the [previous step](../panels/), we focused on the app structure with Panels. Next, we'll investigate how custom and native events are used to notify your app of user actions, how state is defined for the app, and how to use events to update the state of your app. We'll wrap up with a brief discussion on how to manage state data.
 
 ## Events in React
 
-[Native events in React](https://reactjs.org/docs/events.html) are registered using the camelCase version (`onMouseDown`) of their native name (`onmousedown`). If you wish to be notified of an event, you can pass a function as the value for the appropriate prop. The function will receive a synthetic event as its first argument, which is a cross-browser wrapper around the original event.
+[Native events in React](https://react.dev/learn/responding-to-events) are registered using the camelCase version (`onMouseDown`) of their native name (`onmousedown`). If you wish to be notified of an event, you can pass a function as the value for the appropriate prop. The function will receive a synthetic event as its first argument, which is a cross-browser wrapper around the original event.
 
 React doesn't provide an explicit implementation of custom events. In Enact, custom events follow React's event name pattern of having an `on` prefix followed by the name of the event. Like native events, custom events will receive the event payload as the first argument but it will be a simple object, not a synthetic event.
 
 ## Defining our State
 
-For our App, we have two pieces of state we need to manage: the active panel and the selected kitten. Both can be represented as numbers representing the index of the panel and the index of the kitten in `kittens`. We will also define two custom events, `onNavigate` and `onSelectKitten`, that we can use to indicate when each of those indices should change.
+For our App, we have two pieces of state we need to manage: the active panel (`List` or `Detail`) and the selected kitten. Both can be represented as numbers representing the index of the panel and the index of the kitten in `kittens`. We will also define two custom events, `onPanelIndexChange` and `onKittenIndexChange`, that we can use to indicate when each of those indices should change.
```

Comment:
Don't like to use (),
the active panel (`List` or `Detail`) and the selected kitten
-> the active panel, which consists of List and Detail, and the selected kitten.
or just simply
-> the active panel and the selected kitten

Reply:
I accepted your suggestion.
(the active panel, which consists of List and Detail, and the selected kitten.)